File: test-fixtures/swift-grammar/Auth/UserService.swift
import Foundation

public class UserService {
    private let name: String
    var loginCount: Int = 0

    public init(name: String) {
        self.name = name
    }

    public func save(user: String) -> String {
        return user
    }

    public func save(user: String, retries: Int) -> String {
        return user
    }

    class AuditLog {
        func record() {
        }
    }
}

File: test-fixtures/java-grammar/Auth/UserService.java
package com.example.auth;

import java.util.List;
import java.util.Objects;

public class UserService {
    private final String name;
    private int loginCount, failureCount;

    public UserService(String name) {
        this.name = name;
    }

    public String save(String user) {
        return user;
    }

    public String save(String user, int retries) {
        return user;
    }

    class AuditLog {
        void record() {
        }
    }
}
